CONTRIBUTING.md

```diff
@@ -11,7 +11,6 @@ Here are some [instructions][link_signupinstructions].
 Already know what you're looking for in this guide? Jump to the following sections:
 
```

Comment:
This is a very useful section, I think there are a couple of other points in the document readers might want to quickly navigate to, so I would add:
* [installing in editable mode](#Run-the-developer-setup)
* [testing and linting](#Test-your-changes)
* [updating the documentation after changes to the code](#Changes-to-documentation)

I would also link to the developer docs here with a brief description of the contents, say: 
See also our Developing Guidelines on [readthedocs][link_developing_rtd] for additional information, particularly on our monthly developer calls and writing tests for tedana.